### PR TITLE
Adds Discord link to `/content/discuss/_index.md`

### PR DESCRIPTION
**Pull request by johnnymatthews**
Adds a link to the IPFS Discord group, as per: https://github.com/protocol/docs/issues/9

### CODE REVIEW

File: content/discuss/_index.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ We love talking about libp2p, and we'd be happy to have you in the mix.
 
 Visit our discussion forums at [discuss.libp2p.io](https://discuss.libp2p.io) to ask questions about [using libp2p](https://discuss.libp2p.io/c/users), discuss exciting [research](https://discuss.libp2p.io/c/research), and [stay up to date with important announcments](https://discuss.libp2p.io/c/news). 
 
-We also often hang out on IRC, at [#libp2p on irc.freenode.org](irc://irc.freenode.org/%23libp2p). While IRC is a great place for a quick live chat, remember that discussions can get lost in a sea of messages. For long-lived discussion & support, [the discussion forums](https://discuss.libp2p.io) are the way to go.
+We also often hang out on [Discord](https://discord.com/channels/806902334369824788/942671304530747402) and [IRC](irc://irc.freenode.org/%23libp2p). While Discord and IPFS are great places for live chats, remember that discussions can get lost in a sea of messages. For long-lived discussion & support, [the discussion forums](https://discuss.libp2p.io) are the way to go.
```
Comment (mxinden) [inline content/discuss/_index.md:11]:
```suggestion
We also often hang out on [Discord](https://discord.com/channels/806902334369824788/942671304530747402) and [IRC](irc://irc.freenode.org/%23libp2p). While Discord and IRC are great places for live chats, remember that discussions can get lost in a sea of messages. For long-lived discussion & support, [the discussion forums](https://discuss.libp2p.io) are the way to go.
```

Though I wonder whether we want to still list IRC in the first place.

Comment (johnnymatthews) [reply content/discuss/_index.md:11]: This comment from @biglep says we should keep IRC links available. https://github.com/protocol/docs/issues/9#issuecomment-1044646000

**Review by mxinden — APPROVED** [review]
Thanks @johnnymatthews!

**Comment by h1z1** [comment]
Curious was this tested ?  You can't link directly to channels on discord unless you're joined to the server already.  Should use an invite link.

**Pull request merged**